guides/common/modules/ref_resource-quota-settings.adoc

```diff
@@ -0,0 +1,8 @@
+[id="resource-quota-settings"]
+= Resource Quota settings
+
+The Resource Quota plugin contains the following global {Project} settings:
+
+* *Global Resource Quota no action* {endash} {Project} will not limit the resource consumption if a resource quota is exceeded.
+* *Resource Quota optional assignment* {endash} {Project} will not enforce selecting a resource quota during host provisioning.
+The global setting overwrites user-specific and user group-specific configuration options.
```

Comment:
```suggestion
*Global Resource Quota no action*:: {Project} will not limit the resource consumption if a resource quota is exceeded.

*Resource Quota optional assignment*:: {Project} will not enforce selecting a resource quota during host provisioning.
The global setting overwrites user-specific and user group-specific configuration options.
```
I'd go with this formatting for the settings instead. It looks better than an en-dashed list.

Reply:
Applied!